src/test/java/tic/tac/toe/RulesTest.java

```diff
@@ -0,0 +1,82 @@
+package tic.tac.toe;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+/*
+Other potential ideas:
+rules.hasWinner(board); // true / false
+rules.isOver(board); // true / false
+rules.isInProgress(board); // true / false
+rules.isTie(board); // true / false
+
+GameStatus.IN_PROGRESS / TIE_GAME / HAS_WINNER
+GameStatus.IN_PROGRESS / TIE_GAME / PLAYER_ONE_WINS
+ */
+public class RulesTest {
```

Comment:
I would have expected more tests of this class. Ideally, this class would have been fully test-driven before updating the `ComputerPlayer` to use it.

For example, I would have expected tests for winning in a row, a column, a diagonal, etc.